File: Python3/Maximum Number of Words Found in Sentences.py
#1
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        ma = 0
        for s in sentences:
            sp = s.split(" ")
            ma = max(ma, len(sp))

        return ma
        

#2
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        return max(s.count(" ") for s in sentences) + 1
        

File: Golang/Maximum Number of Words Found in Sentences.go
func mostWordsFound(sentences []string) int {
    ma := 0
    for _, s := range sentences {
        slice := strings.Split(s, " ")
        ma = max(ma, len(slice))
    }
    return ma
}

func max(a, b int) int {
    if a < b {
        return b
    }
    return a
}